### PR TITLE
Fix links

### LINKED ISSUE #38: [linkchecker] Review links failing in link checker
This is the latest link checker results that need to be reviewed

https://github.com/QuantEcon/lecture-jax/actions/runs/4909942321/jobs/8766649230

### PR DESCRIPTION
**Pull request by HengchengZhang**
Hi @mmcky, fix #38.

### CODE REVIEW

**Comment by mmcky** [comment]
thanks @HengchengZhang 

```
(  newtons_method: line  137) broken    https://python.quantecon.org/newton_method.html#first_newton_attempt - Anchor 'first_newton_attempt' not found
(kesten_processes: line  146) ok        https://python.quantecon.org/kesten_processes.html#the-kestengoldie-theorem
(  newtons_method: line   43) broken    https://python.quantecon.org/newton_method.html#two_goods_market - Anchor 'two_goods_market' not found
```

Is this the `anchor` error issue you're talking about before (i.e. false positives?)

**Comment by HengchengZhang** [comment]
Hi @mmcky, they are just broken links and all of them should be fixed now.

File: lectures/aiyagari_jax.md [hunk 1]
```diff
@@ -29,16 +29,16 @@ The model features
 
 The Aiyagari model has been used to investigate many topics, including
 
-- precautionary savings and the effect of liquidity constraints [[Aiy94](https://python.quantecon.org/zreferences.html#id137)]
-- risk sharing and asset pricing [[HL96](https://python.quantecon.org/zreferences.html#id129)]
-- the shape of the wealth distribution [[BBZ15](https://python.quantecon.org/zreferences.html#id130)]
+- precautionary savings and the effect of liquidity constraints [[Aiy94](https://python.quantecon.org/zreferences.html#id138)]
```
Comment (mmcky) [inline lectures/aiyagari_jax.md:32]: @HengchengZhang I am going to merge this to fix the links in the short term but we need to migrate all these links to use proper `cite` roles such as: https://manual.quantecon.org/styleguide/references.html

**Comment by mmcky** [comment]
thanks @HengchengZhang for correcting the links. I just had one comment -- perhaps we can open an issue for tracking to do an audit across the lectures to convert some of the `md` links to use `cite` roles.

**Comment by HengchengZhang** [comment]
That's a great idea! I think there could be a lot of similar citations.

**Pull request merged**